lib/ontologies_linked_data/models/mod/semantic_artefact_catalog.rb

```diff
@@ -0,0 +1,228 @@
+            define_attribute :isReferencedBy, namespace: :mod
+            define_attribute :funding, namespace: :mod
+            define_attribute :qualifiedAttribution, namespace: :mod
+            define_attribute :publishingPrinciples, namespace: :mod
+            define_attribute :qualifiedRelation, namespace: :mod
+            define_attribute :fairScore, namespace: :mod
+            define_attribute :featureList, namespace: :mod, enforce: [:list]
+            define_attribute :supportedSchema, namespace: :mod
+            define_attribute :conformsTo, namespace: :mod
+            define_attribute :catalog, namespace: :mod
+            define_attribute :dataset, namespace: :mod
+            define_attribute :service, namespace: :mod
+            define_attribute :record, namespace: :mod
+            define_attribute :themeTaxonomy, namespace: :mod
+            define_attribute :distribution, namespace: :mod
+
+            attribute :modified, namespace: :dcterms, enforce: [:date], handler: :modification_date
+            attribute :numberOfArtefacts, namespace: :mod, enforce: [:integer], handler: :ontologies_count
+            attribute :metrics, namespace: :mod, enforce: [:list], handler: :generate_metrics
+            attribute :numberOfClasses, namespace: :mod, enforce: [:integer], handler: :class_count
+            attribute :numberOfIndividuals, namespace: :mod, enforce: [:integer], handler: :individuals_count
+            attribute :numberOfProperties, namespace: :mod, enforce: [:integer], handler: :propoerties_count
+            attribute :numberOfAxioms, namespace: :mod, enforce: [:integer], handler: :axioms_counts
+            attribute :numberOfObjectProperties, namespace: :mod, enforce: [:integer], handler: :object_properties_counts
+            attribute :numberOfDataProperties, namespace: :mod, enforce: [:integer], handler: :data_properties_counts
+            attribute :numberOfLabels, namespace: :mod, enforce: [:integer], handler: :labels_counts
+            attribute :numberOfDeprecated, namespace: :mod, enforce: [:integer], handler: :deprecated_counts
+            attribute :numberOfUsingProjects, namespace: :mod, enforce: [:integer], handler: :using_projects_counts
+            attribute :numberOfEndorsements, namespace: :mod, enforce: [:integer], handler: :endorsements_counts
+            attribute :numberOfMappings, namespace: :mod, enforce: [:integer], handler: :mappings_counts
+            attribute :numberOfUsers, namespace: :mod, enforce: [:integer], handler: :users_counts
+            attribute :numberOfAgents, namespace: :mod, enforce: [:integer], handler: :agents_counts
+            
+            attribute :test_attr_to_persist
+            serialize_never :test_attr_to_persist
+
+            serialize_default :acronym, :title, :color, :description, :logo, :fundedBy, :versionInfo, :homepage, :numberOfArtefacts, :federated_portals
+
+            def bring(*attributes)
```

Comment:
no needed of this. use regular bring.

Reply:
this is because the regular bring, will not bring the computed attributes if we don't put them as serialize_default, the one that has the tag :handler like this one

```ruby
attribute :numberOfObjectProperties, namespace: :mod, enforce: [:integer], handler: :object_properties_counts
```
so i did this bring to bring the other computer attributes without put them as serialize_default

Reply:
Fix the bring method in Goo to bring computed attributes even if not set as serialized by default